courses/utils.py

```diff
@@ -30,6 +33,42 @@ def is_grade_valid(override_grade: float):
     return 0.0 <= override_grade <= 1.0
 
 
+def has_earned_program_cert(user, program):
```

Comment:
Could you actually move this and the next two methods into `api.py` instead? They definitely fit in there more than in `utils.py`

Reply:
moved